docker-compose.yml

```diff
@@ -23,12 +23,18 @@ volumes:
 services:
   redis:
     image: "redis:alpine"
+    healthcheck:
+      test: [CMD, redis-cli, ping]
 
   memcached:
-    image: memcached
+    image: memcached:alpine
+    healthcheck:
+      test: [CMD, nc, -z, localhost, '11211']
 
   postgres:
-    image: "postgres:11"
+    image: postgres:11-alpine
```

Comment:
Not necessary for the healthcheck, but should be a harmless change, so might as well.